src/routes/Ecosystem/Common/VaultField.tsx

```diff
@@ -0,0 +1,67 @@
+import { IconLabel } from '@components/composite';
+import { Typography } from '@components/atomic';
+import { Box } from '@mui/system';
+import React from 'react';
+import { titleStyles } from './styles';
+import { ReactComponent as Mellow } from '../mellow-icon.svg';
+
+type VaultFieldProps = {
+  title: string;
+  expectedApy: string;
+  maturity: string;
+};
+
+export const VaultField: React.FunctionComponent<VaultFieldProps> = ({
+  title,
+  expectedApy,
+  maturity,
+}: VaultFieldProps) => {
+  return (
+    <Box>
+      <Box sx={{ display: 'flex', marginLeft: '8px', marginTop: '16px', alignItems: 'center' }}>
```

Comment:
can you follow same guidelines for styling components we introduced in the Profile? Create a file called VaultField.styled.tsx and there you export your styled components. You can check any .styled.tsx file for idea. Please follow this across project.